spec/integrations/shared_examples.rb

```diff
@@ -72,7 +72,7 @@
       ws = Faye::WebSocket::Client.new(ws_url)
 
       ws.onopen = lambda do |event|
-        ws.send('{"message_sequence":0}')
+        ws.send('{"last_message_sequence":0}')
```

Comment:
What do you think about a test for the deprecated behavior, so we make sure we don't break it until a major version upgrade?